dbms/src/Common/FailPoint.h

```diff
@@ -66,6 +68,8 @@ class FailPointHelper
      */
     static void initRandomFailPoints(Poco::Util::LayeredConfiguration & config, const LoggerPtr & log);
 
+    static void disableRandomFailPoints(Poco::Util::LayeredConfiguration & config, const LoggerPtr & log);
```

Comment:
Better add some comments to describe, it is just a "clear" operation.

Reply:
ok, comment `3. Call disableFailPoint method with parsed FailPointName and ignore Rate.` added.